examples/reference-entity-model/steps/mapping/mapCustomersWithRelatedEntitiesJSON.step.json

```diff
@@ -0,0 +1,179 @@
+
+{
+  "collections": ["mapCustomersWithRelatedEntitiesJSON", "Customer"],
+  "additionalCollections": [],
+  "permissions": "data-hub-common,read,data-hub-common,update",
+  "batchSize": 100,
+  "validateEntity": "doNotValidate",
+  "targetFormat": "JSON",
+  "name": "mapCustomersWithRelatedEntitiesJSON",
+  "targetEntityType": "http://example.org/Customer-0.0.1/Customer",
+  "selectedSource": "collection",
+  "sourceQuery": "cts.collectionQuery(['loadCustomersWithRelatedEntitiesJSON'])",
+  "sourceDatabase": "data-hub-STAGING",
+  "targetDatabase": "data-hub-FINAL",
+  "headers": {},
+  "interceptors": [],
+  "provenanceGranularityLevel": "coarse",
+  "customHook": {},
+  "stepDefinitionName": "entity-services-mapping",
+  "stepDefinitionType": "mapping",
+  "stepId": "mapCustomersWithRelatedEntitiesJSON-mapping",
+  "properties": {
+    "customerId": {
+      "sourcedFrom": "CustomerID"
+    },
+    "name": {
+      "sourcedFrom": "concat(Name/FirstName, ' ', Name/LastName)"
+    },
+    "shipping": {
+      "sourcedFrom": "/Address/Shipping",
+      "properties": {
+        "street": {
+          "sourcedFrom": "Street"
+        },
+        "city": {
+          "sourcedFrom": "City"
+        },
+        "state": {
+          "sourcedFrom": "State"
+        },
+        "zip": {
+          "sourcedFrom": ".",
+          "properties": {
+            "fiveDigit": {
+              "sourcedFrom": "substring(Postal, 1, 5)"
+            },
+            "plusFour": {
+              "sourcedFrom": "substring(Postal, 7)"
+            }
+          },
+          "targetEntityType": "#/definitions/Zip"
+        }
+      },
+      "targetEntityType": "#/definitions/Address"
+    },
+    "billing": {
+      "sourcedFrom": "/Address/Billing",
+      "properties": {
+        "street": {
+          "sourcedFrom": "Street"
+        },
+        "city": {
+          "sourcedFrom": "City"
+        },
+        "state": {
+          "sourcedFrom": "State"
+        },
+        "zip": {
+          "sourcedFrom": ".",
+          "properties": {
+            "fiveDigit": {
+              "sourcedFrom": "substring(Postal, 1, 5)"
+            },
+            "plusFour": {
+              "sourcedFrom": "substring(Postal, 7)"
+            }
+          },
+          "targetEntityType": "#/definitions/Zip"
+        }
+      },
+      "targetEntityType": "#/definitions/Address"
+    },
+    "customerSince": {
```

Comment:
Remove these properties from the mapping too, just to simplify the mapping as much as possible so we can focus on related entities.